pkg/core/writer/tracetracker/tracker.go

```diff
@@ -98,68 +99,136 @@ func (a *ActiveServiceTracker) processSpan(span *trace.Span, now time.Time) {
 	if span.LocalEndpoint == nil || span.LocalEndpoint.ServiceName == nil {
 		return
 	}
-
 	service := *span.LocalEndpoint.ServiceName
-	a.ensureServiceActive(service, now)
+
+	// TODO: check on what the tag name should be in order to fetch environment
+	environment, environmentFound := span.Tags["environment"]
+	if !environmentFound {
+		// TODO set to default environment
```

Comment:
Yes, we are going to set a default environment for both pod/container and host-level.  For host dims we want to send that default always because there are multiple environments.  But for k8s/container dims, we have a plan to send the default and then overwrite it if a real value comes in on a span tag.